server.js

```diff
@@ -322,6 +382,51 @@ app.get('/:username', async (req, res) => {
 });
 
 
+// Edit user profile
+app.patch("/:username/edit", authenticateUser);
+app.patch("/:username/edit", async (req, res) => {
+  try {
+    const accessToken = req.header("Authorization");
+    const user = await User.findOne({ accessToken: accessToken });
+    const username = req.params.username;
+    if (user.username === username) {
+      const { username, email, password } = req.body;
```

Comment:
we have email here as well

Reply:
This is so you can edit your email address, but we can also remove the entire email property.